content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -31,6 +41,17 @@ as the Kubernetes control plane, is responsible for all communication with the r
 To configure a KMS provider on the API server, include a provider of type `kms` in the
 `providers` array in the encryption configuration file and set the following properties:
 
+### KMS v1
```

Comment:
I do not think it makes sense to separate these by API version since we have the `apiVersion` field.

Reply:
I think we should separate them because `apiVersion` field is only available after v1.25 and we do want to add it to this doc as that is required to be set to v2 if you are using v2. wdyt?